vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -221,23 +222,14 @@ public void setVimeoAccount(@Nullable VimeoAccount vimeoAccount) {
     }
 
     /**
-     * @deprecated use {@link #saveAccount(VimeoAccount, String)} instead
-     * <p/>
-     * We find no use in storing the password when you can persist the {@link VimeoAccount} across
-     * application sessions.
-     */
-    @Deprecated
-    public void saveAccount(@Nullable VimeoAccount vimeoAccount, String email, String password) {
-        saveAccount(vimeoAccount, email);
-    }
-
-    /**
      * Sets the {@link #mVimeoAccount} field as well as triggering the saveAccount event for the
      * account store
      */
     public void saveAccount(@Nullable VimeoAccount vimeoAccount, String email) {
```

Comment:
I ignored email because it should be passed in when there is an authenticated user token, but can be null when there is a client credentials grant. I will add javadoc to the matter, since I think an annotation for Nullable is misleading.

Also, we allow a null account to be passed in since the methods fetching the account could return null, and this method will correctly construct one if so.